samples/net/openthread/ncp/overlay-tri-n4m-br.conf

```diff
@@ -14,7 +14,7 @@ CONFIG_STDOUT_CONSOLE=y
 CONFIG_GPIO=y
 CONFIG_USB=y
 CONFIG_USB_DEVICE_STACK=y
-CONFIG_USB_DEVICE_PRODUCT="OT BorderRouter K64"
+CONFIG_USB_DEVICE_PRODUCT="OpenThread NCP K64"
```

Comment:
This file, which is for a board that " The board support is not part of the Zephyr repositories, ...can be used as an example", is not necessary anymore because `overlay-usb-nrf-br.conf` can be used as an example now? Can you remove it then?

Reply:
I'd keep it. It can be used with FRDM-K64 + RF233 shield.

Reply:
We do not have support for a RF233 shield, please remove this file.

Reply:
True, there is no shield in the tree, but the RF233 driver is included in the tree.
Can remove the file, just want to make sure that it's really wanted.

Reply:
@jfischer-phytec-iot do you really want to have the file removed? or is it OK to keep as an example?

Reply:
yes, because overlay-usb-nrf-br.conf can be used as example now and we do not have any shields with RF233.

Reply:
well, now @jukkar has merged in the meantime.

Reply:
That is okay, I did not block this PR. You could cleanup next time you touch this sample.